src/lib/darkcoinClient.ts

```diff
@@ -154,6 +139,31 @@ export default class DarkcoinClient {
 
   // GObjects
 
+  /**
+   * The gobject prepare RPC prepares a governance object by signing and creating a collateral transaction.
+   * @param parentHash Hash of the parent object. usually the root node which has a hash of 0
+   * @param revision Object revision number (Always 0)
+   * @param creationTime Creation time as a unix timestamp
+   * @param gobjectData Object data (JSON object with governance details)
+   * @returns Transaction id for the collateral transaction
+   */
+  public gobjectPrepare(parentHash: string, revision: number, creationTime: number, gobjectData: string): Promise<CallResult<string>> {
+    return this.callRPCMethod<string>('gobject', ['prepare', parentHash.toString, revision.toString, creationTime.toString, gobjectData]);
+  }
+
+  /**
+   * The gobject submit RPC submits a governance object to network (objects must first be prepared via gobject prepare).
+   * @param parentHash Hash of the parent object. Usually the root node which has a hash of 0
+   * @param revision Object revision number (Always 0)
```

Comment:
Same comment regarding wondering about the value of this parameter

Reply:
Revision should be `1` ^